packages/payload/src/uploads/generateFileData.ts

```diff
@@ -318,7 +322,7 @@ export const generateFileData = async <T>({
  */
 function parseUploadEditsFromReqOrIncomingData(args: {
   data: unknown
-  operation: 'create' | 'update'
+  operation: 'create' | 'duplicate' | 'update'
```

Comment:
This should be left alone to be consistent with our existing `operations` type (even though it isn't referenced here).